dm/dm.go

```diff
@@ -96,18 +111,39 @@ func main() {
 		usage()
 	}
 
+	command := args[0]
 	if *stdin {
-		fmt.Printf("reading from stdin is not yet implemented")
-		os.Exit(0)
+		if len(args) != 2 {
+			usage()
+		}
+
+		cmd := exec.Command("tar", "xf", "-")
```

Comment:
I'm sure it works fine, but it seems much cleaner to use a library rather than an executable, no? Seems like it would be a cleaner, and we wouldn't have to change the entire error handling to panics?